doc/tutorials/core/mat_the_basic_image_container/mat_the_basic_image_container.markdown

```diff
@@ -210,7 +210,7 @@ object in multiple ways:
 
     @note
     You can fill out a matrix with random values using the @ref cv::randu() function. You need to
-    give the lower and upper value for the random values:
+    give the minimum and the maximum value for the random values:
```

Comment:
This is slightly misleading: `randu` accepts inclusive lower boundary and exclusive upper boundary parameters. "_maximum value_" implies inclusive upper boundary.

Reply:
Fixed in a9d23a6